pkgs/servers/gemini/agate/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, rustPlatform, installShellFiles }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "agate";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "mbrubeck";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-rwoEZnxh0x+xaggJuoeSjE1ctF43ChW5awcDJyoWioA=";
+  };
+
+  cargoSha256 = "sha256-ey/fUHkPoWjWlLjh1WNpwMKOkdQKgFYcLwQdx2RQ3CI=";
+
+  meta = with lib; {
+    homepage = "gemini://gem.limpet.net/agate";
+    changelog = "gemini://gem.limpet.net/agate";
```

Comment:
```suggestion
    homepage = "https://proxy.vulpes.one/gemini/gem.limpet.net/agate";
    changelog = "https://proxy.vulpes.one/gemini/gem.limpet.net/agate";
```

Reply:
I guess I can put the github but for some reason theres no changelog on there
But since this is a gemini server I'd expect the person to be happy using that protocol.

Same with the gopher one, its specifically a gopher terminal browser